tools/mocha/src/mocha.js

```diff
@@ -35,6 +36,9 @@ const runCLI = () => {
   global.stub = global.sinon.stub;
   global.assert = global.sinon.assert;
   global.stubModule = require('./lib/stubber');
+  global.setEventuallyPollingInterval = setEventuallyPollingInterval;
```

Comment:
It seems a big awkward to set a global polling interval. It changes the global per-module defaults. Only the fact that we execute tests in new process allows us to have different timeouts.

Having per call config makes this more explicit:
```
const lenghtyPollingConfig = {
  pollingInterval: 10000
}

const shortPollingConfig = {
  pollingInterval: 100
}

eventually(func1, shortPollingConfig)
eventually(func2, shortPollingConfig)
eventually(func2, lengthyPollingConfig)
```

I wonder can we make this more scope-bounded, like mocha test timeouts.

Reply:
Yes, this is true. But if we want to set the configuration once and execute all functions with the same config.